cve_bin_tool/cvedb.py

```diff
@@ -584,6 +589,8 @@ def populate_affected(self, affected_data, cursor, data_source):
         except Exception as e:
             LOGGER.info(f"Unable to insert data for {data_source} - {e}")
 
+    """Adding data to metric table."""
+
     def populate_metrics(self):
```

Comment:
```suggestion

    def populate_metrics(self):
        """ Adding data to metric table. """
```

Same deal.  Beware, I'm doing suggestions from the web interface so my indentation may be incorrect if you try to just merge these in.